Dockerfile

```diff
@@ -1,13 +1,13 @@
 FROM golang:1.9.2-alpine3.6 AS build
-
+ARG BRANCH=master
```

Comment:
Consider changing branch to commit.

Reply:
We will review it in the future

Reply:
As it will cause issues w/ the default master